android/library/core/src/main/java/com/liferay/mobile/screens/assetlist/interactor/AssetListInteractorImpl.java

```diff
@@ -116,13 +119,16 @@ protected void getPageRowsRequest(Session session, int startRow, int endRow, Loc
 
 	@Override
 	protected void getPageRowCountRequest(Session session) throws Exception {
-		JSONObject entryQueryParams = addQueryParams(_groupId, _classNameId);
+		JSONObject entryQueryParams = configureEntryQuery(_groupId, _classNameId);
 		JSONObjectWrapper entryQuery = new JSONObjectWrapper(entryQueryParams);
 		new AssetEntryService(session).getEntriesCount(entryQuery);
 	}
 
-	protected JSONObject addQueryParams(long groupId, long classNameId) throws JSONException {
-		JSONObject entryQueryParams = new JSONObject();
+	protected JSONObject configureEntryQuery(long groupId, long classNameId) throws JSONException {
+
+		JSONObject entryQueryParams =
+			_customEntryQuery == null ? new JSONObject() : new JSONObject(_customEntryQuery);
+
```

Comment:
We shouldn't overwrite custom values with the screenlet's classNameId, groupId or visible